bokehjs/src/coffee/models/layouts/layout_dom.coffee

```diff
@@ -121,7 +121,7 @@ class LayoutDOMView extends BokehView
 class LayoutDOM extends Model
   type: "LayoutDOM"
 
-  constructor: (attrs, options) ->
+  initialize: (attrs, options) ->
```

Comment:
Interesting! I had a lot of back and forth on this. I tried to always use initialize where I could. 

But sometimes it seemed to be necessary to use constructor. I may have been wrong in this case or this was an artefact when I was juggling too many balls in the layout PR, but just wanted to check in on this.

If I'm recalling correctly, which I may not be, it was because constraints started getting added to the solver before the model was fully initialized...but I'm feeling pretty hazy. Anyway, I expect it would just blow up if this was meant to be constructor.